deps.bzl

```diff
@@ -384,3 +384,19 @@ genrule(
 )
             """,
         )
+
+    if "canton" not in native.existing_rules():
+        http_archive(
+            name = "canton",
+            build_file_content = """
+        package(default_visibility = ["//visibility:public"])
+
+        java_import(
+            name = "lib",
+            jars = glob(["lib/**/*.jar"]),
+        )
```

Comment:
```suggestion
package(default_visibility = ["//visibility:public"])

java_import(
    name = "lib",
    jars = glob(["lib/**/*.jar"]),
)
```

Reply:
with that it wroks locally for me so hopefully it also fixes CI

Reply:
Thanks - let's give it a try.

Reply:
seems to work now, you’re missing an (empty) changelog entry

Reply:
of course